op-batcher/batcher/driver.go

```diff
@@ -945,3 +920,134 @@ func logFields(xs ...any) (fs []any) {
 	}
 	return fs
 }
+
+type ChannelStatuser interface {
+	isFullySubmitted() bool
+	isTimedOut() bool
+	LatestL2() eth.BlockID
+	MaxInclusionBlock() uint64
+}
+
+var SeqOutOfSyncError error = errors.New("sequencer out of sync")
```

Comment:
In idiomatic Go, only error _types_ are `FooError`s. This should be
```suggestion
var ErrSeqOutOfSync error = errors.New("sequencer out of sync")
```

Reply:
[SeqOutOfSyncError -> ErrSeqOutOfSync](https://github.com/ethereum-optimism/optimism/pull/13060/commits/1ed9fff79abbc9347c34c794f1d64951c27a9b59)